pkg/kubeletconfig/kubeletconfig.go

```diff
@@ -46,3 +49,12 @@ func KubeletConfToMCKubeletConf(kcObj *kubeletconfigv1beta1.KubeletConfiguration
 	kcAsMc.Spec.KubeletConfig.Raw = rawKc
 	return err
 }
+
+func DecodeFromData(data []byte, scheme *runtime.Scheme, mcoKc *mcov1.KubeletConfig) error {
```

Comment:
nit: best would be to fix the API so it would look like
```
func DecodeFromData(data []byte, scheme *runtime.Scheme) (*mcov1.KubeletConfig, error) {
}
```